### ISSUE TITLE
Docker/Traefik OData returns wrong URL

### ISSUE THREAD

**Issue by jpreniers** [microsoft/nav-docker#460]
We are using BC Docker containers with Traefik proxy server, but we cannot use the Odata URL from the BC Containers in Power-BI desktop or Excel. For example; the officially Odata URL from the container is: https://xxxxxxxxxxxxxxxxx.com/ENTERPRISErest/ODataV4/  but if we open this url in a browser we see that it’s return a wrong url: https://xxxxxxxxxxxxxxxxx.com/BC/ODataV4. So the BC instance is returned wrong. 
The PublicODataBaseUrl Is set correctly in the BC NST configuration.

**Comment by freddydk** [comment]
Yes, that is correct. If anything, it is a bug in the Business Central platform that the PublicODataBaseUrl isn't applied to the output. I think I filed a bug a long time ago, will check up what the status is, it isn't really tied to traefik, but traefik exposes it.
In the meantime, you will have to replace the start of the url with the publicodatabaseurl yourself.

**Comment by babaczyk** [comment]
It's BC14, PublicOdataBaseUrl looks properly, but data feed address in the OData is wrong. Sorry but maybe I didn't understand your response - am I able to fix that feed address presented in OData?

![image](https://user-images.githubusercontent.com/29023561/96553030-df051e80-12b4-11eb-9a0e-c3335719ba76.png)

**Comment by freddydk** [comment]
It is a bug in the NAV/BC platform - not something I can fix in Docker

**Comment by babaczyk** [comment]
Thanks, I understand this is a BC error, but I asked rather that if I can change that value by myself? Because I only know PublicODataBaseUrl parameter of the Service Tier configuration. Maybe you know if there is anything else I can configure to force the correct feed address? Thanks in advance.

**Comment by freddydk** [comment]
Unfortunately not

**Comment by freddydk** [comment]
I heard that this should be fixed in the latest version of BC, I have not tried myself.
